Modules/_io/clinic/_iomodule.c.h

```diff
@@ -272,6 +272,52 @@ _io_open(PyObject *module, PyObject *const *args, Py_ssize_t nargs, PyObject *kw
     return return_value;
 }
 
+PyDoc_STRVAR(_io_text_encoding__doc__,
+"text_encoding($module, encoding, stacklevel=2, /)\n"
+"--\n"
+"\n"
+"Helper function to choose the text encoding.\n"
+"\n"
+"When encoding is not None, just return it.\n"
+"Otherwise, return the default text encoding (i.e. \"locale\").\n"
+"\n"
+"This function emits EncodingWarning if *encoding* is None and\n"
+"sys.flags.warn_default_encoding is true.\n"
+"\n"
+"This function can be used in APIs having encoding=None option.\n"
+"But please consider encoding=\"utf-8\" for new APIs.");
```

Comment:
```suggestion
"A helper function to choose the text encoding.\n"
"\n"
"When encoding is not None, just return it.\n"
"Otherwise, return the default text encoding (i.e. \"locale\").\n"
"\n"
"This function emits an EncodingWarning if *encoding* is None and\n"
"sys.flags.warn_default_encoding is true.\n"
"\n"
"This can be used in APIs with an encoding=None parameter.\n"
"However, please consider using encoding=\"utf-8\" for new APIs.");
```

Same as above; not sure if this is generated or not

Reply:
This file is generated from `_iomodule.c`.

Reply:
Gotcha, thanks. Feel free to resolve.